notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    EXPLODE_PIE_CHART = tuple([0.05] * LIMIT)
</pre><p>What is this .05 value? Why did you choose it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
No reason. This constant separates pie chart to distinguish each part more visible and to look good.

<img width="430" alt="image" src="https://user-images.githubusercontent.com/38146012/179317651-319f3020-db2f-4265-b81c-0bebe6b37ca5.png">
without exploding

<img width="438" alt="image" src="https://user-images.githubusercontent.com/38146012/179317540-a2307e0a-256e-4734-995b-f315c34a84be.png">
with exploding

The number `.05` is interval among pies heuristically determined.